content/blog/2023/unintuitive-objects.md

```diff
@@ -0,0 +1,476 @@
+have an image (with the title included),
+but a SERIES has seasons and episodes
+while a MOVIE has a total length.
+Even if I wanted a consistent rectangle card
+on my homepage,
+I could at least include the number of seasons
+on the SERIES card
+and the total length
+on the MOVIE card.
+Even that small difference in the designs
+of a MOVIE and a SERIES object
+would go a long way toward unmasking these objects.
+To make a MOVIE and a SERIES
+even more instantly differentiated,
+I could do something like
+make all MOVIE cards square
+and all SERIES cards rectangular.
+
+Now, it is arguable that MOVIES and SERIES
+aren't actually different objects in this system.
+They do have a large amount of the same attributes.
+Perhaps they are two different types
+of a VIDEO (There's probably a better word.) object.
+That decision is up to the UX designer,
+but even in that case,
+it would be a more intuitive experience,
+for me anyway,
+to see a MOVIE or a SERIES tag
+on these cards.
+
+### Shapeshifter Objects
+*Objects that are styled to look different, but are actually the same.*
+
+In my experience,
+one of the most common reasons
+a website vistor will experience a site
+as unintuitive
+is due to shapeshifter objects.
+Not only is avoiding shapeshift objects
+as huge win for the user,
+it can also save UX designers a lot of time
+creating mockups.
+
+Let's look again at the DEMO object again
+on the zoo website.
+This object actually appears
+in at least two different list views
+on the website.
+These lists both display DEMOS,
+but for reasons that are unclear to me
+the DEMO object looks different
+in each list.
+One list
+shows each DEMO title once
+with all the times the DEMO happens
+right under the title.
+The other list
+shows each DEMO title multiple times
+with only one of the times the DEMO happens.
+In the first list the times are in bold.
+In the other list the times are italicized.
+In the first list the am and pm do not have punctuation
+and in the other they do.
+The first list does not include
+the DEMO location.
+The other does.
+
+{{ embed.img(
+  src='blog/2023/masked-demo.jpg',
+  alt='two versions of the demo list with different styles'
+) }}
+
+These are fairly minor differences.
+If you start to look for shapeshifters yourself,
+you will find much more dramatic redesigns
+of the same object
+for no discernable reason.
+Still, even with these small differences,
+as a visitor to the website,
+it is not immediately obvious
+that these two lists contain instances
+of exactly the same DEMO object.
+Thinking back to the physical world of objects,
+how disconcerting and unintuitive would it be
+if you encountered a chair object
+at a desk and a chair object at a table,
+but in the context of the desk
+the chair had four legs,
+and in the context of a the table
+the chair had three legs.
+You would definitely have to
+spend some time
+thinking through whether the chair with three legs
+would work for you as a chair!
```

Comment:
I think I'd remove this entire section? I'm not sure it helps much, and honestly 3-legged chairs are a real thing and I don't think they're very confusing. 😅

Reply:
I agree. Don't chairs come in all different shapes and sizes that are still pretty recognizable and usable…